Tests/Packet++Test/Tests/Asn1Tests.cpp

```diff
@@ -21,9 +21,10 @@ PTF_TEST_CASE(Asn1DecodingTest)
 		PTF_ASSERT_EQUAL(record->getValueLength(), 11);
 		PTF_ASSERT_EQUAL(record->toString(), "ContextSpecific (7), Length: 2+11");
 		auto genericRecord = record->castAs<pcpp::Asn1GenericRecord>();
-		auto recordValue = std::string(genericRecord->getValue(), genericRecord->getValue() + genericRecord->getValueLength());
+		auto recordValue =
+		    std::string(genericRecord->getValue(), genericRecord->getValue() + genericRecord->getValueLength());
 		PTF_ASSERT_EQUAL(recordValue, "objectclass");
-	}
+	};
```

Comment:
Why is this `;` added here? 🤔

Reply:
Without `;`, clang-format will have wrong formatting.  I added it intentionally.

Reply:
clang-format has some weird bugs... 😕

Reply:
@seladb  let's keep this change?

Reply:
yes, we can keep it